maintainers/maintainer-list.nix

```diff
@@ -2171,6 +2171,16 @@
     githubId = 4971975;
     name = "Janne Heß";
   };
+  dasisdormax = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add dasisdormax`` and sort it alphabetically.